packages/experimental/kai/src/containers/TaskList.tsx

```diff
@@ -154,17 +162,45 @@ export const TaskItem: FC<{
   task: Task;
   readonly?: boolean;
   showAssigned?: boolean;
-  onEnter?: (task: Task) => void;
   onDelete?: (task: Task) => void;
   onSave?: (task: Task) => void;
-}> = withReactor(({ task, readonly, showAssigned, onEnter, onDelete, onSave }) => {
+  isLast?: boolean;
+  orderIndex: number;
+}> = withReactor(({ task, readonly, showAssigned, onDelete, onSave, orderIndex, isLast }) => {
   const { debug } = useOptions();
   useReactorContext({
     onChange: () => {
       onSave?.(task);
     }
   });
 
+  const onKeyUp = useCallback(
+    (e: KeyboardEvent<HTMLInputElement>) => {
+      switch (e.key) {
+        case 'Enter':
+          if (!e.shiftKey) {
+            if (isLast) {
+              (document.querySelector('input#new-task') as HTMLElement | undefined)?.focus();
+            } else {
+              (
+                document.querySelector(`input[data-orderindex="${orderIndex + 1}"]`) as HTMLElement | undefined
+              )?.focus();
+            }
+          }
+          break;
+        case 'PageDown':
+          e.preventDefault();
```

Comment:
this didn't seem to be working for me, it shifted the focus but if there was scroll possible on the page it would also trigger that as well

Reply:
also, it's not currently possible to pagedown into the new task input which felt a little awkward in practice